tests/importable_test.py

```diff
@@ -185,6 +185,6 @@ def test_entry_points_are_credential_plugin(entry_point: str) -> None:
 def test_entry_points_are_inventory_plugin(entry_point: str) -> None:
```

Comment:
I think I missed this in between rounds of refactoring the PR that introduced the tests. The actual argument type in all the tests is `EntryPointParam`. “Parametrize” used to be structured differently and inject strings into tests, before I changed the data struct but forgot to replace `str` here...